src/main/drivers/gyro_sync.c

```diff
@@ -53,3 +71,20 @@ uint8_t gyroMPU6xxxGetDividerDrops(void)
 {
     return mpuDividerDrops;
 }
+
+void gyroProcessingCallback(void) {
```

Comment:
This is not really a callback. In my view the term "callback" is overused. In my view a call back is a function pointer you pass in a procedure call, that the callee calls to signal something, hence call-back.

Whereas this is just a function call.

I'd prefer something like `gyroHandleInterrupt`